### DISCUSSION TITLE
blog/2024/10/29/VS-Code-Thankyou/

### DISCUSSION THREAD

**Discussion by giscus** [imbant/blog#20] [General]
# blog/2024/10/29/VS-Code-Thankyou/

imbant 前端工程师技术博客

https://imbant.github.io/blog/2024/10/29/VS-Code-Thankyou/

**Comment by msdone-lwt** [comment]
如果像学习 lsp 的话，大佬有什么推荐的文章或者视频吗

**Reply by imbant** [reply]
Hi，刚看到评论。lsp 是个小众领域，我在入门时苦于资料太少，所以想自己写一些。
英文资料的话，推荐你去看 vscode 语言插件开发的相关文档，lsp 本来也是微软的技术，在自家的代码编辑器里支持很好。文档你都过一遍，用例写一遍，基本就对语言客户端有所了解了；然后简单翻翻 lsp 的官方文档，对协议也大概有个了解。vscode 插件文档也有专门讲到语言服务器开发，这样整个架构就串起来了。
中文资料的话，你可以参考我的其他博文，以及关于我里的即刻里也会写一些平时学习和开发的想法。厚着脸皮说一句，也许 lsp 的中文资料里我的是最好的😝

**Reply by msdone-lwt** [reply]
好的 谢谢

**Reply by imbant** [reply]
Hi 系列教程写好了 https://imbant.github.io/blog/2024/08/24/LSP1/

**Reply by msdone-lwt** [reply]
大佬 牛逼，幸苦了，写的非常好
